Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -185,6 +185,22 @@ public extension Attribute where Context == HTML.VideoSourceContext {
     }
 }
 
+public extension Attribute where Context: HTMLSourceSetContext {
+    /// Assign a source to the element, using its `srcset` attribute.
+    /// - parameter url: The source URL to assign.
+    static func srcset(_ url: URLRepresentable) -> Attribute {
+        Attribute(name: "srcset", value: url.string)
+    }
+}
+
+public extension Attribute where Context: HTMLMediaContext {
+    /// Assign a source to the element, using its `media` attribute.
+    /// - parameter url:Any valid media query that would normally be defined in a CSS
```

Comment:
The parameter name should be `mediaQuery` here (although I think you could just call it `query`, since the fact that it's for "media" is already established by the method name). Also please leave a space after the colon 🙂

Reply:
Also, another nitpick (sorry 😅): At the end of the sentence, cut "a": `defined in CSS.`.

Reply:
Will fixed both :)